### DISCUSSION TITLE
Custom Subject DN OID 2.4.5.97 OrganizationIdentifier

### DISCUSSION THREAD

**Discussion by ThePeeBee** [Keyfactor/ejbca-ce#323] [Q&A]
Hi Community,

i am using a complete setup of EJBCA on a CENTOS 7 machine. Installed ejbca version is 6.15.2.6.
What i try to do is, adding a custom subject DN field. I found some workarounds stating, that the /src/java/profilemappings.properties file should be edited. 
i added
DN;2.5.4.97;200;2.5.4.97;200;OrganizationIdentifier;DN_ETSI_ORGANIZATION_IDENTIFIER 
to the bottom line of the file. I also tried this variant:
DN;2.5.4.97;200;2.5.4.97;200;OrganizationIdentifier;OrganizationIdentifier
None of them worked. I still find the basic SubjectDNs in the Dropdownbox of the enduser entity Profile. 
Also i tried to restart the Server after every edit. 
Can someone help me out?

Cheers,

Patrick

**Comment by primetomas** [comment]
OrganizationIdentifier is a standard feature in Enterprise. I know that's not the answer you look for. 
With the method you use you could try using the APIs (WS or REST) instead of looking for it in the UI.
